openapi: @vertz/openapi/adapters export missing from dist (0.1.6)

## Version

`@vertz/openapi@0.1.6`

## Description

The `package.json` declares an `./adapters` export:

```json
"exports": {
  "./adapters": {
    "import": "./dist/adapters/index.js",
    "types": "./dist/adapters/index.d.ts"
  }
}
```

But the `dist/adapters/` directory doesn't exist in the published package:

```
dist/
  cli.d.ts
  cli.js
  index.d.ts
  index.js
  shared/
```

### Reproduction

```ts
// openapi.config.ts
import { fastapi } from "@vertz/openapi/adapters";
```

```
error: Cannot find module '@vertz/openapi/adapters' from 'openapi.config.ts'
```

Looks like the adapters directory wasn't included in the build output.

--- a/.changeset/fix-openapi-adapters-export.md
+++ b/.changeset/fix-openapi-adapters-export.md
@@ -1,0 +1,5 @@
+---
+'@vertz/openapi': patch
+---
+
+Fix `@vertz/openapi/adapters` export missing from dist. Add bunup config with all entry points (index, cli, adapters).

--- a/packages/openapi/bunup.config.ts
+++ b/packages/openapi/bunup.config.ts
@@ -1,0 +1,8 @@
+import { defineConfig } from 'bunup';
+
+export default defineConfig({
+  entry: ['src/index.ts', 'src/cli.ts', 'src/adapters/index.ts'],
+  format: ['esm'],
+  dts: { inferTypes: true },
+  clean: true,
+});

